resources/assets/components/utilities/SitewideBanner/SitewideBanner.js

```diff
@@ -7,9 +7,10 @@ import SitewideBannerContent from './SitewideBannerContent';
 const isExcludedPath = pathname => {
   return excludedPaths.find(excludedPath => {
     if (excludedPath.includes('*')) {
+      const pathWithoutAsterisk = excludedPath.slice(0, -1);
       return (
         pathname.includes(excludedPath.slice(0, -1)) &&
```

Comment:
Would be sweet to use the `pathWithoutAsterisk` here as well!
```suggestion
        pathname.includes(pathWithoutAsterisk) &&
```